policyengine_us/parameters/gov/states/ny/nysdps/nys_clean_heat/conE_incentive.yaml

```
@@ -0,0 +1,82 @@
+description: New York State Clean Heat rebate (conEdison)
+metadata:
+  unit: currency-USD #TODO: Includes both integer and strings. Need to be fixed.
+  breakdown:
+    - clean_heat_conEdison_classification
+  period: year
+  label: New York State Clean Heat rebate (conEdison)
+  reference:
+    - title: New York State Clean Heat Con Edison Heat Pump Program Manual
+      href: https://cleanheat.ny.gov/assets/pdf/CECONY%20Clean%20Heat%20Program%20Manual%206%203%2024.pdf
+
+'Residential':
+    'ASHP':
+        'Non-DAC':
+            'Single Family Home':
+                'ccASHP with integrated controls':
+                    2024-03-01: 2_500
+                'AWHP with decomissioning':
+                    2024-03-01: 8_000
+            'Apartment':
+                'ccASHP with decommissioning':
+                    2024-03-01: 1_000
+                'AWHP with decomissioning':
+                    2024-03-01: 4_000
+        'DAC':
+            'Single Family Home':
+                'ccASHP with integrated controls':
+                    2024-03-01: 4_500
+                'AWHP with decomissioning':
+                    2024-03-01: 10_000
+            'Apartment':
+                'ccASHP with decommissioning':
+                    2024-03-01: 2_000
+                'AWHP with decomissioning':
+                    2024-03-01: 5_000
+    'GSHP':
+        'Non-DAC':
+            2024-03-01: 25_000
+        'DAC':
+            2024-03-01: 35_000
+"Multifamily":
+    "GSHP":
+        "New Construction":
+            "Multifamily Full Load ASHP Heating with Decommissioning":
+                2024-03-01: "N/A"
+            "Custom Space Heating Applications + Envelope -- Tier 1":
+                2024-03-01:  "125/MMBtu"
+            "Custom Space Heating Applications + Envelope -- Tier 2": 
+                2024-03-01: "150/MMBtu"
+            "Custom Domestic Hot Water": 
+                2024-03-01: "125/MMBtu"
+            "Prescriptive Domestic Hot Water": 
+                2024-03-01: "N/A"
+            "Custom Partial Load Space Heating Applications": 
+                2024-03-01: "N/A"
+        "Existing Building":
+            "Multifamily Full Load ASHP Heating with Decommissioning": 
+                2024-03-01: "5000/dwelling"
+            "Custom Space Heating Applications + Envelope -- Tier 1": 
+                2024-03-01: "200/MMBtu"
+            "Custom Space Heating Applications + Envelope -- Tier 2": 
+                2024-03-01: "225/MMBtu"
+            "Custom Domestic Hot Water": 
+                2024-03-01: "200/MMBtu"
+            "Prescriptive Domestic Hot Water": 
+                2024-03-01: "N/A"
+            "Custom Partial Load Space Heating Applications": 
+                2024-03-01: "100/MMBtu"
+    "ASHP":
+        "Existing Building":
```

```suggestion
        EXISTING:
```
We will use the ENUM value in the breakdown as opposed to the description 
E.g. here we use "EXISTING" instead of "Existing Building"